pandas/_libs/tslib.pyx

```diff
@@ -5713,3 +5454,32 @@ cdef _calc_julian_from_U_or_W(int year, int week_of_year,
 
 # def _strptime_time(data_string, format="%a %b %d %H:%M:%S %Y"):
 #     return _strptime(data_string, format)[0]
+
+#----------------------------------------------------------------------
+# Parsing
+# Wrap tslibs.parsing functions to return `NaT` instead of `NAT_SENTINEL`
+
+
+def parse_time_string(arg, freq=None, dayfirst=None, yearfirst=None):
```

Comment:
why is this here? it should be in ``tslib.parsing``, if you *need* ``NaT`` *inside* the function there, simply import it (inside the function)

Reply:
Was trying to avoid runtime import; changed in just-pushed update.